Makefile

```diff
@@ -45,6 +46,9 @@ ks:
 docs:
 	$(DOC_GEN_FILE)
 
+docker-image:
+	docker build -t ks --build-arg LD_FLAGS="$(LD_FLAGS)" .
```

Comment:
We should probably tag according to the ksonnet version as well. Maybe `git describe --tags` with or without the `v` prefix?

Reply:
if you look in the LD_FLAGS it grabs `git rev-parse HEAD` which gives you sha1 ... but do you mean for human consumption like having a ks images  named ks-0.11.0 or something along those lines?

Reply:
Yeah look how images are tagged on DockerHub (where these images should ultimately be pushed via Jenkins). I'd only include sha for non-release tags.